src/plugins/intel_gpu/src/graph/impls/onednn/fully_connected_onednn.cpp

```diff
@@ -360,6 +382,16 @@ struct fully_connected_onednn : typed_primitive_onednn_impl<fully_connected> {
                 }
             }
 
+            if (prim->dynamic_quantized_activation) {
+                // Note: it supports per-token activation scale only
+                ++idx;
+                auto partial_shape = impl_params.input_layouts[0].get_partial_shape();
+                auto innermost_len = partial_shape[partial_shape.size() - 1].get_length();
+
+                auto act_scale_data_type = convert_data_type(impl_params.input_layouts[idx].data_type);
+                attr->set_scales(DNNL_ARG_SRC, (1 << 1) | (1 << 0), dnnl::memory::dims{1, innermost_len}, act_scale_data_type);
```

Comment:
1. If that's per token case, then shouldn't mask be equal to `2`?
2. Could you please add some macro wrappers for these masks ? like PER_TENSOR=1, PER_CHANNEL=2, GROUPED=3

Reply:
When I checked benchdnn, it did not support mask 2 for activation scale. That is why we are using 2.

Reply:
`(1 << 1) | (1 << 0)` the meaning of this bitmask is unclear, so we'd better have some macro for different values to improve code readability

Reply:
applied, thanks!